EpikCord/__init__.py

```diff
@@ -17,7 +17,7 @@
 from importlib import import_module
 from inspect import iscoroutine
 from logging import getLogger
-from time import perf_counter_ns
+import time
```

Comment:
...why?

Reply:
?

Reply:
Not sure why you changed `from time import perf_counter_ns` to `import time`

Reply:
Same.